app/controllers/agentAppointment/AgentRelationshipController.scala

```diff
@@ -78,10 +78,10 @@ class AgentRelationshipController @Inject()(
       },
       success => {
         for {
-          organisationsAgents <- agentRelationshipService.getMyOrganisationAgents()
-          agentOpt            <- agentRelationshipService.getAgent(success.toLong)
+          organisationsAgents    <- agentRelationshipService.getMyOrganisationAgents()
```

Comment:
why are you getting this?

Reply:
im getting it because I need to determine if the agent has already been added to the org